.circleci/test-artifacts.sh

```diff
@@ -4,4 +4,10 @@
 #
 set -e
 
+# Create screenshots directory in case it was not created before. This is to
```

Comment:
Is this just something that didn't get merged?

Reply:
Ah, I've used `master` as the base branch. I'll update it to `develop` now.